app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -241,6 +242,21 @@ class DeveloperOptionsActivityTest {
     }
   }
 
+  @Test
+  fun testDeveloperOptionsFragment_clickForceCrash_assertException() {
+    launch<DeveloperOptionsActivity>(
+      createDeveloperOptionsActivityIntent(
+        internalProfileId = internalProfileId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      assertThrows(RuntimeException::class) {
```

Comment:
Please verify the message of the exception being thrown to ensure the correct runtime exception is being thrown.

Reply:
Done.